lib/project_types/script/commands/enable.rb

```diff
@@ -37,10 +33,6 @@ def call(args, _name)
       def self.help
         ShopifyCli::Context.message('script.enable.help', ShopifyCli::TOOL_NAME)
       end
-
-      def self.extended_help
```

Comment:
I've added two options (`--config_file` and `--config_props`), so the extended help is still required here.

Reply:
I think whichever PR merges in first, the subsequent PR will need to rebase.  It's a race!!  🚗   😄